docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapKeyToVector.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic
+{
+    class MapKeyToVector
+    {
+        /// This example demonstrates the use of MapKeyToVector by mapping keys to floats[]. 
```

Comment:
Ah... here's another one I found. This cherry pick (commit id eb57d09) supposedly corresponds to #3287, but it includes content that was introduced originally in #3211, which should have been cherry picked from this.